### PR TITLE
Fix support for explicit TLS

### PR DESCRIPTION
**Pull request by preng69**

### CODE REVIEW

**Comment by lunny** [comment]
seems good. have you tested that?

**Comment by preng69** [comment]
I've tested with FileZilla, FireFTP and curl. Server configured for explicit TLS on port 21. Observations with other clients are welcome!

FireZilla: explicit TLS and passive mode is default, so the "Quick connect" now works without any settings, just hostname.

FireFTP: Security has to be changed to "Auth TLS" from default "None", otherwise defaults.

Curl: "curl --ftp-ssl -u USERNAME:PASSWORD ftp://HOSTNAME" works.

File: cmd.go [hunk 1]
```diff
@@ -928,7 +928,7 @@ func (cmd commandPbsz) RequireParam() bool {
 }
 
 func (cmd commandPbsz) RequireAuth() bool {
-	return true
+	return false
```
Comment (lunny) [inline cmd.go]: Why change this to `false`?

**Comment by lunny** [comment]
Hi, could you rebase and add some tests since we have a test framework now?

**Pull request merged**